extensions/google-chrome/src/util/index.ts

```diff
@@ -1,26 +1,36 @@
+import fs from "fs";
 import path from "path";
 import { DEFAULT_CHROME_PROFILE_ID, defaultChromeProfilePath, defaultChromeStatePath } from "../constants";
 import { getPreferenceValues } from "@raycast/api";
 import { Preferences } from "../interfaces";
 
+type ChromeFile = "History" | "Bookmarks";
 const userLibraryDirectoryPath = () => {
   if (!process.env.HOME) {
     throw new Error("$HOME environment variable is not set.");
   }
 
   return path.join(process.env.HOME, "Library");
 };
-const getChromeFilePath = (fileName: string, profile?: string) => {
+const getChromeFilePath = (fileName: ChromeFile, profile?: string) => {
   const { profilePath } = getPreferenceValues<Preferences>();
+  let resolvedProfilePath;
   if (profilePath) {
-    return path.join(profilePath, fileName);
+      resolvedProfilePath =path.join(profilePath, fileName);
   }
-  return path.join(
+  else {
+    resolvedProfilePath= path.join(
     userLibraryDirectoryPath(),
     ...defaultChromeProfilePath,
     profile ?? DEFAULT_CHROME_PROFILE_ID,
     fileName
-  );
+  );}
+
+  if (!fs.existsSync(resolvedProfilePath)) {
+    throw new Error(`The path ${resolvedProfilePath} does not exist. Please check your Chrome profile location by visiting chrome://version -> Profile Path.`);
```

Comment:
Looks good.  Just needs prettier run in order to pass the tests.

One small addition to the message. Those experiencing the error won't always know that profile customization exists now (yay!), so a hint pointing them to the setting will likely help them resolve the problem efficiently.


```suggestion
    throw new Error(`The path ${resolvedProfilePath} does not exist. Please check your Chrome profile location by visiting chrome://version -> Profile Path. Then update it in Extension Settings -> Profile Path.`);
```